docs/walkthrough_collaborate.md

```diff
@@ -2,19 +2,72 @@
 title: Collaborate within eScriptorium
 summary: About features related to collaboration in eScriptorium.
 authors:
-    - Alix Chagué
-date: 2023-01-31
+    - Hugo Scheithauer, Alix Chagué
+date: 2023-02-09
 ---
 
-# Walkthrough : collaborate within eScriptorium
+# Walkthrough: collaborate within eScriptorium
 
-!!! warning
-    The section needs to be completed.
+This walkthrough details features related to collaboration in eScriptorium.
 
-## Create a team or join a team
+## Teams
+
+Users can be grouped by Teams. The main purpose of a team is to facilitate [sharing documents or projects](#share-a-project-or-a-document). Teams are also helpful to identify a group of people working on the same project or coming from the same institution.  
+
+It is possible to manage teams from the [Profile page](walkthrough_users.md). Team management includes:  
+
+- Creating a team,  
+- Leaving a team,  
+- Adding or removing members from a team,  
+- Transfering ownership to another member.  
+
+Only the owner of a team is allowed to add or remove members or to transfer ownership. The owner of a team cannot leave it without first transferring the ownership to another user.
+
+??? Note "No deleting, no renaming"
+    It is currently impossible to delete a team or to rename a team.
+
+??? Note "One owner per team"
+    There can only be one owner at a time.
 
 ## Invite a user
 
-## Share a project or a document 
+eScriptorium allows you to invite new users to your instance.
+
+You can find more information about inviting new users in the [user-related features walkthrough](walkthrough_users.md#Invite) section.
+
+## Share a project or a document
+
+You can share projects and documents to others users of your eScriptorium instance.
+
+### Share a project
+
+You can share an opened project you own by clicking on the "Share this project" button. It will then open a pop-up box where you have two options:
+
+- A form in which you can enter any username existing on your eScriptorium instance,
```

Comment:
I would precise that their is no auto-completion for username in that field, you should simply be in touch with a user by other means in order to know their username on that instance. Also, it doesn't work if you put their email address or their full name.

Reply:
If you made enter an incorrect username, the application will likely still consider that sharing was successful. So one way to make sure that the project is really shared with the users you want, is to click again on "share this project": they will appeared in the list of Users, with the box next to them checked. 

There is probably a simpler way to explain that..